Documentation/feature-band-source-building.md

```diff
@@ -0,0 +1,893 @@
+# .NET VMR Feature Band Source Building Guide for Linux Distro Maintainers
+
+This document provides guidance for Linux distribution maintainers on source
+building .NET SDK feature band branches of the Virtual Monolithic Repository
+(VMR). It covers bootstrapping and ongoing servicing workflows when building
+multiple SDK feature bands.
+
+## Table of Contents
+
+- [What are .NET SDK Feature Bands?](#what-are-net-sdk-feature-bands)
+- [Key Terminology](#key-terminology)
+- [Overview](#overview)
+- [Understanding Bootstrap vs Sequential Build](#understanding-bootstrap-vs-sequential-build)
+- [Required Build Inputs](#required-build-inputs)
+- [Feature Band Characteristics](#feature-band-characteristics)
+- [Build Requirements by Feature Band](#build-requirements-by-feature-band)
+- [Build Command Arguments](#build-command-arguments)
+- [Distro Maintainer Workflows](#distro-maintainer-workflows)
+  - [Input Artifacts Summary](#input-artifacts-summary)
+  - [1xx Band Bootstrap](#1xx-band-bootstrap)
+  - [1xx Band Servicing](#1xx-band-servicing)
+  - [2xx Band Initial Release (N.0.200)](#2xx-band-initial-release-n0200)
+  - [2xx Band Bootstrap (N.0.200+)](#2xx-band-bootstrap-n0200)
```

Comment:
My inclination is to list >= 2xx bootstrap scenarios as last.  They would be an edge case where the initial and servicing are canonical usage scenarios.